instrumentation/jetty-8.0/src/main/java/io/opentelemetry/auto/instrumentation/jetty/JettyHandlerAdvice.java

```diff
@@ -86,7 +95,14 @@ public static void stopSpan(
 
     // Check again in case the request finished before adding the listener.
     if (!request.isAsyncStarted() && responseHandled.compareAndSet(false, true)) {
+      contentLengthHelper(span, response);
       TRACER.end(span, response.getStatus());
     }
   }
+
+  public static void contentLengthHelper(Span span, ServletResponse response) {
+    if (response instanceof CountingHttpServletResponse) {
```

Comment:
Notice JettyHandler didn't seem to have implemented counting, so thought it'd be a quick fix to add it, but some conflict between unnamed classloader and app classloader here :( Tomorrow will give another small look and then revert this and remove the assertion for now.

Reply:
Found it was because I was missing this advice from the helper classes (was referring to servlet-3.0 while writing this). Filed #796 since this was confusing.